frontend/src/libs/components/app/app.tsx

```diff
@@ -27,6 +29,7 @@ const App: React.FC = () => {
 
   return (
     <>
+      <Header user={null} />
```

Comment:
When auth is implemented, do not forget to change this passed value user from the store